client/src/app/shared/header/header.component.ts

```diff
@@ -10,6 +10,7 @@ export class HeaderComponent implements OnInit {
   constructor() { }
 
   ngOnInit() {
+    console.debug('test');
```

Comment:
__[TSLint]__

Calls to 'console.debug' are not allowed. (no-console)


<!-- Comment by [Sider](https://sider.review) -->